proto/cosmos/tx/v1beta1/tx.proto

```diff
@@ -116,6 +116,14 @@ message TxBody {
   // be processed by the chain
   uint64 timeout_height = 3;
 
+  // non_atomic is an atomicity flag. If false, the transaction will be atomic
+  // and be rejected if any of the messages fail. This is the default and
+  // compatible with previous versions of the SDK and TxBody.
+  //
+  // If set to true, the transaction will be executed as long as at least one
+  // of the messages succeeds.
```

Comment:
Should we clarify here, that only state changes from succeeding msgs will persist?

Reply:
yeah, will do

Reply:
Kind ping @nicolaslara!

Reply:
Kind ping @nicolaslara!